src/mmw/js/src/draw/views.js

```diff
@@ -67,6 +67,7 @@ function validateShape(polygon) {
         message += Math.floor(area) + ' square km were selected, ';
         message += 'but the maximum supported size is currently ';
         message += MAX_AREA + ' square km.';
+        window.alert(message);
```

Comment:
I think it was accidentally lost here https://github.com/WikiWatershed/model-my-watershed/commit/210891039ea37cbd16c684660d62d9a9f7a94890#diff-72c102beb3924e611bfc083303b1f8d9L69